
const ValidateData=async(req,res)=>{
    try{

    }
    catch(error){
        
    }
}